### ISSUE TITLE
Searching gets stuck after some point

### ISSUE THREAD

**Issue by savan77** [keras-team/autokeras#492]
I am trying to find the best model for cats-dogs dataset with 20000 training and 5000 testing images. Everything was running fine, but after epoch-13 it got stuck and nothing was happening. I even checked system usage, GPU was not being used. I also checked logs but there isn't anything helpful. Here is the last part of logs.

`
Epoch-12, Current Metric - 0.85:   0%|                                    | 0/4 [00:00<?, ? batch/s]
Epoch-12, Current Metric - 0.85: 10 batch [00:00, 49.54 batch/s]                                    
                                                                

Epoch-13, Current Metric - 0.888:   0%|                                 | 0/153 [00:00<?, ? batch/s]
Epoch-13, Current Metric - 0.888:   7%|█▌                      | 10/153 [00:02<00:33,  4.28 batch/s]
Epoch-13, Current Metric - 0.888:  13%|███▏                    | 20/153 [00:06<00:38,  3.47 batch/s]
Epoch-13, Current Metric - 0.888:  20%|████▋                   | 30/153 [00:10<00:40,  3.06 batch/s]
Epoch-13, Current Metric - 0.888:  26%|██████▎                 | 40/153 [00:14<00:39,  2.83 batch/s]
Epoch-13, Current Metric - 0.888:  33%|███████▊                | 50/153 [00:18<00:38,  2.68 batch/s]
Epoch-13, Current Metric - 0.888:  39%|█████████▍              | 60/153 [00:23<00:35,  2.59 batch/s]
Epoch-13, Current Metric - 0.888:  46%|██████████▉             | 70/153 [00:27<00:32,  2.53 batch/s]
Epoch-13, Current Metric - 0.888:  52%|████████████▌           | 80/153 [00:31<00:29,  2.49 batch/s]
Epoch-13, Current Metric - 0.888:  59%|██████████████          | 90/153 [00:35<00:25,  2.46 batch/s]
Epoch-13, Current Metric - 0.888:  65%|███████████████        | 100/153 [00:39<00:21,  2.44 batch/s]
Epoch-13, Current Metric - 0.888:  72%|████████████████▌      | 110/153 [00:43<00:17,  2.43 batch/s]
Epoch-13, Current Metric - 0.888:  78%|██████████████████     | 120/153 [00:48<00:13,  2.42 batch/s]
Epoch-13, Current Metric - 0.888:  85%|███████████████████▌   | 130/153 [00:52<00:09,  2.41 batch/s]
Epoch-13, Current Metric - 0.888:  92%|█████████████████████  | 140/153 [00:56<00:05,  2.41 batch/s]
Epoch-13, Current Metric - 0.888:  98%|██████████████████████▌| 150/153 [01:00<00:01,  2.41 batch/s]
Epoch-13, Current Metric - 0.888: 160 batch [01:04,  2.41 batch/s]                                  
                                                                  

Epoch-13, Current Metric - 0.888:   0%|                                   | 0/4 [00:00<?, ? batch/s]
Epoch-13, Current Metric - 0.888: 10 batch [00:00, 49.12 batch/s]                                   
                                                                 
No loss decrease after 5 epochs.

Saving model.
+--------------------------------------------------------------------------+
|        Model ID        |          Loss          |      Metric Value      |
+--------------------------------------------------------------------------+
|           34           |   1.222847518324852    |   0.8671999999999999   |
+--------------------------------------------------------------------------+

+----------------------------------------------+
|              Training model 35               |
+----------------------------------------------+

Epoch-1, Current Metric - 0:   0%|                                      | 0/153 [00:00<?, ? batch/s]`

**Comment by nmelgiri** [comment]
I'm getting the same issue; it might be a bug.

**Comment by ineersa** [comment]
I'm getting the same issue

**Comment by brigittee** [comment]
I'm getting the same issue. It got stuck at training model 37.
